src/api/entities/Subsidy/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Account, Context, Entity, quitSubsidy } from '~/internal';
+import { NoArgsProcedureMethod } from '~/types';
+import { createProcedureMethod, toHumanReadable } from '~/utils/internal';
+
+export interface UniqueIdentifiers {
+  beneficiaryAddress: string;
+  subsidizerAddress: string;
```

Comment:
I think `beneficiary` and `subsidizer` are enough here. We can explain they're addresses with JSDoc comments ("beneficiary address", "subsidizer address")